appyters/Augmented_Pathway_Enrichment_Analysis/requirements.txt

```diff
@@ -0,0 +1,15 @@
+git+git://github.com/Maayanlab/appyter.git
+git+git://github.com/Maayanlab/maayanlab-bioinformatics.git
+pandas
+matplotlib
+seaborn
+numpy
+bokeh
+pybase64
+statsmodels
+csv
+itertools
+json
+requests
+time
```

Comment:
`time`, `json`, `csv`, `itertools`, `urllib` are not valid for `requirements.txt`; only modules which you've `pip install`-ed are; the ones I listed are in the python standard library and will cause an error.